content/en/docs/distributions/gke/cloud-filestore.md

```diff
@@ -2,15 +2,9 @@
 title = "Using Cloud Filestore"
 description = "Using Cloud Filestore with Kubeflow"
 weight = 60
-                    
-+++
```

Comment:
Note, it's related to the title.
Removing this "+++" caused your tests to fail.

Reply:
Thank you for spotting it! Fixed.